### PR TITLE
feature/ja-45_Add logic for methods

### PR DESCRIPTION
**Pull request by skrawus**

### CODE REVIEW

**Comment by skrawus** [comment]
some methods still in progress

**Review by GrzegorzKmiecinski — APPROVED** [review]
looks good to me

File: TutorLizard.BusinessLogic/Data/DataAccess.cs [hunk 1]
```diff
@@ -199,47 +199,63 @@ private int GetNewScheduleItemRequestID()
...
     public void UpdateAdRequest(AdRequest adRequest)
     {
-        throw new NotImplementedException();
+        int index = _adRequestList.FindIndex(ar => ar.Id == adRequest.Id);
+
+        if (index > 0)
+        {
+            _adRequestList[index] = adRequest;
+        }
+        throw new Exception("Not found");
```
Comment (GrzegorzKmiecinski) [inline TutorLizard.BusinessLogic/Data/DataAccess.cs]: This will throw an exception with every method call

Comment (skrawus) [reply TutorLizard.BusinessLogic/Data/DataAccess.cs]: I've changed the name of the exception, but actually not suer if this will work just fine

**Review by GrzegorzKmiecinski — APPROVED** [review]

**Pull request merged**